Traceback (most recent call last):
  File "/home/ubuntu/group_1_work/technical_assistant/error_check_file_07.py", line 3, in <module>
    import nonexistent_module
ModuleNotFoundError: No module named 'nonexistent_module'
